nativeApp/src/screens/admin_views/Volunteers/VolunteersPage/index.tsx

```diff
@@ -19,27 +27,73 @@ type Props = {
  */
 
 /*
-* Component
-*/
+ * Component
+ */
+const Volunteers: FC<NavigationFocusInjectedProps & Props> = ({ navigation }) => {
+  // Redux
+  const dispatch = useDispatch();
 
+  useEffect(() => {
+    dispatch(loadVolunteers());
```

Comment:
So since we're now loading volunteers on another page, this API call is potentially wasteful. What we want is to understand whether the volunteers that are in our store are outdated. We can add a key `stale` to the state, which is marked `false` by default, but marked `true` if we perform any action which might cause the entity/entities to change on the backend. Then we can check this value before performing another API call. This check could even be part of the thunk (because the component shouldn't really have to care about it).

I think we can make this change in a subsequent PR, rather than needing to do it in this one.

Reply:
How about loading all entity requests on app load, then any update and deletion actions can manually call a reload after they return?

Reply:
That doesn't seem like it would scale well -- slowing down initial load for data that you might not need is not great. We can discuss this when the next PR is started.

Reply:
Yeah, that's a fair point